crates/executor/src/executor.rs

```diff
@@ -166,7 +167,13 @@ where
                         match db.contracts.entry(account.info.code_hash) {
                             Entry::Vacant(entry) => {
                                 entry.insert(code.clone());
-                                new_bytecodes.insert(H256(account.info.code_hash.0), code.clone());
+                                new_bytecodes.insert(
+                                    H256(account.info.code_hash.0),
+                                    // TODO: We have to analyse the bytecode again here, leading to
+                                    // double analysis since revm does not propagate analysed
+                                    // bytecode up to the database.
+                                    to_analysed(code.clone()),
```

Comment:
The analysis done by revm is not given to the database. There is an open issue for it https://github.com/bluealloy/revm/issues/390

Reply:
It can be made so created bytecode is analyzed, and it is by default like that: https://github.com/bluealloy/revm/blob/7bb73da23d0278829f9669f175a6eede247c0538/crates/primitives/src/env.rs#L89
This issue is only if loaded bytecode is raw and we would like to transform it to analysed, this needs to be done.